server/core/config/valid/global_cfg.go

```diff
@@ -232,41 +195,47 @@ func NewGlobalCfgFromArgs(args GlobalCfgArgs) GlobalCfg {
 		},
 	}
 
-	if args.PlatformModeEnabled {
-		globalCfg.WorkflowMode = PlatformWorkflowMode
+	globalCfg.Repos = []Repo{repo}
 
-		// defaultPullRequstWorkflow is only used in platform mode. By default it does not
-		// support apply stage, and plan stage run with -lock=false flag
-		pullRequestWorkflow := Workflow{
-			Name:        DefaultWorkflowName,
-			Plan:        DefaultLocklessPlanStage,
-			PolicyCheck: DefaultPolicyCheckStage,
-		}
+	return globalCfg
+}
 
-		deploymentWorkflow := Workflow{
-			Name:  DefaultWorkflowName,
-			Apply: DefaultApplyStage,
-			Plan:  DefaultPlanStage,
-		}
+func (g GlobalCfg) EnablePolicyChecks() GlobalCfg {
+	repo := &g.Repos[0]
+	repo.ApplyRequirements = append(repo.ApplyRequirements, PoliciesPassedApplyReq)
+	return g
+}
```

Comment:
Would be dope at some point if we could remove this and dynamically support policy checks if the config/policies are present.